include/zephyr/dt-bindings/gpio/nordic-nrf-gpio.h

```diff
@@ -38,17 +38,22 @@
 
```

Comment:
Please update comment above to align with current implementation.

Reply:
Done :)

Reply:
How about also adding in line 22:
`Extra high (E) drive mode can be used in SoCs that support it. It has to be applied to both pin levels.`

Reply:
Certainly :)